src/index.js

```diff
@@ -296,6 +296,72 @@ exports.init = function(app, config) {
       .subscribe(observeToResponse(res, ' '));
     });
 
+    /* GET /repo/:repo/show-branch?rev=<rev>[&rev=<rev>...]
+    *
+    * Response:
+    *   json: [ ({ "rev": <rev>, "commits": [ ({ "sha": <sha>, "message": <message> })* ] })* ]
```

Comment:
Does this also return just the commits? Doesn't log (the below) already do that?

Reply:
Actually the behavior of show-branch and log can differ in some cases (which are not entirely clear to me actually). In the end we could not achieve the diff between two branches only with show-branch.